var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -129,6 +129,11 @@ def determine_version(cls, exe):
         return match.group(1) if match else None
 
     def setup_build_environment(self, env):
+        if self.spec.satisfies('@:8.0.61'):
+            # Perl 5.26 removed current directory from module search path,
+            # CUDA 9 has a fix for this, but CUDA 8 and lower don't.
+            env.set('PERL5LIB', self.stage.source_path)
```

Comment:
Do you think `env.set` is save or should we `env.append_path`?

Reply:
Sure, it looks better. Sorry I am not very familiar with Spack's API.